sdks/python/src/honcho/utils/file_upload.py

```
@@ -0,0 +1,75 @@
+        elif isinstance(file_content, IOBase):  # pyright: ignore -- needed for return type
+            return (filename, file_content, content_type)
+        else:
+            raise ValueError("File content must be bytes or a file-like object.")
+
+    # If it's a file object (not str/bytes/bytearray/memoryview)
+    elif isinstance(file, IOBase):  # pyright: ignore -- needed for return type
+        filename = getattr(file, "name", None)
+        if not filename:
+            raise ValueError("File object must have a .name attribute.")
+        content_type = mimetypes.guess_type(filename)[0] or "application/octet-stream"
+        return (filename, file, content_type)
```

Ruff (0.12.2)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)